src/helpers/quit-app.ts

```diff
@@ -0,0 +1,16 @@
+import { sendNativeKeyboardEvent } from './send-keyboard-event';
+import { wait } from '../utils/wait';
+import { appState } from '../renderer/state';
+import { getRunningSlackProcessesCount } from './get-running-slack-processes';
+export async function quit() {
+  await sendNativeKeyboardEvent({
+    text: 'q',
+    cmdOrCtrl: true
+  });
+  // Wait until we have no processes
+  await wait(500);
+  const runningSlackProcessesCount = await getRunningSlackProcessesCount(
+    appState
+  );
+  runningSlackProcessesCount === 0;
```

Comment:
Make the user know if the helper has failed to quit all Slack processes by raising an error.
```suggestion
  if (runningSlackProcessesCount > 0) {
    throw new Error('Failed to quit Slack from quit helper!');
  }
```